src/Routes.js

```diff
@@ -28,7 +28,7 @@ const paths = {
 const InsightsRoute = ({ component: Component, rootClass, ...rest }) => {
     const root = document.getElementById('root');
     root.removeAttribute('class');
-    root.classList.add(`page__${rootClass}`, 'pf-l-page__main');
```

Comment:
No need to remove this line, we need to keep both classes there untill chrome PR gets merged.